src/app/pages/contact-page/contact-page.component.scss

```diff
@@ -106,4 +106,7 @@ a {
   margin-top: 20px;
 }
 
-
+.py-2 {
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
```

Comment:
50px is not the correct value for py-2.
I made a _mixins file in the styles directory. 
Now we can use px-somevalue or py-somevalue in the project, where somevalue is between 1 and 10 and it will apply the padding: 16 * that value. 
Check the comments of the _mixins folder for more details on how to use them.
You can then delete this declaration here and just use something like py-5. 
The padding should be somewhere around 70